Kulyk_Lab/app/src/main/java/com/nazar/kulyk_lab/activities/MainActivity.java

```diff
@@ -1,38 +1,59 @@
 package com.nazar.kulyk_lab.activities;
 
-import android.os.Build;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
-import android.support.annotation.RequiresApi;
+import android.support.v4.widget.SwipeRefreshLayout;
 import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.util.Log;
 import android.view.View;
+import android.widget.TextView;
 
 import com.nazar.kulyk_lab.R;
+import com.nazar.kulyk_lab.adapters.RecyclerViewAdapter;
 import com.nazar.kulyk_lab.interfaces.RijksmuseumApi;
 import com.nazar.kulyk_lab.models.ArtList;
 import com.nazar.kulyk_lab.models.artObjects.ArtObjects;
 
-import java.lang.annotation.Target;
 import java.util.ArrayList;
 import java.util.Objects;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
 import retrofit2.Call;
 import retrofit2.Callback;
 import retrofit2.Response;
 import retrofit2.Retrofit;
 import retrofit2.converter.gson.GsonConverterFactory;
 
 public class MainActivity extends AppCompatActivity {
-    private static String TAG = "MainActivity";
+    @BindView(R.id.recyclerView)
+    RecyclerView recyclerView;
+    @BindView(R.id.swipeContainer)
+    SwipeRefreshLayout swipeContainer;
+    @BindView(R.id.no_data)
+    TextView no_data;
+    private LinearLayoutManager linerLayoutManager = new LinearLayoutManager(this);
+    private RecyclerViewAdapter adapter = new RecyclerViewAdapter();
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        ButterKnife.bind(this);
+        getData();
+        swipeContainer.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                refresh();
+                no_data.setVisibility(View.INVISIBLE);
+            }
+        });
+        swipeContainer.setColorSchemeResources(R.color.colorPrimary);
     }
 
-    private void getData() {
+    public void getData() {
```

Comment:
1. This method is not used anywhere except this class, so it has to be private
2. You should use methods with `get` prefix only for methods that actually return something. In this case, it is `void`, so it should be named differently.